libraries/AP_Compass/AP_Compass.h

```diff
@@ -546,6 +558,15 @@ friend class AP_Compass_Backend;
 
     AP_Int16 _offset_max;
 
+    // Angle difference between compass' in xyz plane, in degrees
+    AP_Float _ang_diff_xyz_max;
+
+    // Angle difference between compass' in xy plane, in degrees
+    AP_Float _ang_diff_xy_max;
+
+    // Mag field Length difference between compass' in xy plane, in mG
+    AP_Float _len_diff_xy_max;
```

Comment:
Are these ever updated?  If not, should these be made `const`?

Can you add units (e.g. `_ang_diff_xyz_max_deg`)?

Reply:
I have made the desired changes. Thanks! for the feedback